### PR TITLE
[CMLG-015]  The toggle button of 10 rows and all rows in the search page. 

### PR DESCRIPTION
**Pull request by ywu666**
Issue:
Try to add a toggle button to the search page that allowed the user to choose what kind of format they want to display the translation table. One is 10 rows per page, the other one is showing all rows.

Solution: 
By using the button group that provided on the bootstrap library and using some css to make the toggle button have the same line as the other components like search Bar and selectCol button. And also make the colour consistence with the search page.

Risk: 
No risk currently.

Reviewed By: 
Emily, Dave, Kevin, Eileen and Annie.

### CODE REVIEW

File: src/components/SearchPage.js [hunk 1]
```diff
@@ -77,8 +87,9 @@ class SearchPage extends React.Component {
     retrieveTableData() {
 
         let sequenceTime = new Date();
-        let url = 'https://cmlgbackend.wdcc.co.nz/api/translations?sequence=' + sequenceTime.getTime() +
-                  '&word=' + this.state.word;
+
+		let url = 'https://cmlgbackend.wdcc.co.nz/api/translations?sequence=' + sequenceTime.getTime() +
```
Comment (annithinggoes) [inline src/components/SearchPage.js:91]: indenting?

File: src/components/SearchPage.js [hunk 2]
```diff
@@ -77,8 +87,9 @@ class SearchPage extends React.Component {
...
+				  '&word=' + this.state.word + "&pageRow=" + this.state.rowsPerPage;
```
Comment (annithinggoes) [inline src/components/SearchPage.js:92]: word can sometimes be empty, wat happens then? default for pagerow is 10 anyways?

File: src/components/css/RowsPerPageToggleButton.css [hunk 3]
```diff
@@ -0,0 +1,10 @@
+.toggle-button {
+    border-radius: unset !important;
```
Comment (annithinggoes) [inline src/components/css/RowsPerPageToggleButton.css:2]: is it a good idea to do !important?

**Comment by zyan225** [comment]
Trying to solve conflicts and make changes to the url now.

**Comment by ywu666** [comment]
> I'm trying to solve conflicts and make changes to the URL now.

**Comment by ywu666** [comment]
@zyan225 When you solve the conflicts of css

File: src/components/css/RowsPerPageToggleButton.css (hunk 3 above)
Comment (zyan225) [reply src/components/css/RowsPerPageToggleButton.css:2]: the default toggle-button here has a round border, has to use important to change that. 
Maybe it's not a good idea... but I don't know if there is another way to get around this.

**Comment by zyan225** [comment]
Pull request updated with master now, can someone review it?

File: src/components/css/RowsPerPageToggleButton.css (hunk 3 above)
Comment (ywu666) [reply src/components/css/RowsPerPageToggleButton.css]: I solve this.

**Comment by ywu666** [comment]
@zyan225 I open the pull request. I can't approve of it. However, you can.

File: src/components/SearchPage.js [hunk 4]
```diff
@@ -18,11 +19,14 @@ class SearchPage extends React.Component {
             isTableLoading: true,
             sequenceNumber: "",
             totalPages: 1,
-            currentPage: 1
+            currentPage: 1,
+			rowsPerPage: 10
```
Comment (EileenSiqi) [inline src/components/SearchPage.js:23]: indentation looks a bit weird here

**Review by EileenSiqi — APPROVED** [review]

File: src/components/SearchPage.js (hunk 4 above)
Comment (ywu666) [reply src/components/SearchPage.js:23]: Strange looks fine on my local.

Comment (annithinggoes) [reply src/components/SearchPage.js:23]: Alsp strange for me

**Review by annithinggoes — APPROVED** [review]

**Pull request merged**